data/campaigns/The_South_Guard/scenarios/07b_Pebbles_in_the_Flood.cfg

```diff
@@ -290,21 +337,42 @@
         first_time_only=no
 
         [filter]
-            side=2
+            side=1
         [/filter]
         [gold]
-            side=2
+            side=2,3
 #ifdef EASY
-            amount=50
+            amount=9
 #endif
 #ifdef NORMAL
-            amount=65
+            amount=12
 #endif
 #ifdef HARD
-            amount=80
+            amount=15
 #endif
         [/gold]
     [/event]
+    [event]
+        name=die
+        first_time_only=no
+
```

Comment:
This scenario already has such an event, with values of 50, 70 gold.
As there are no changes listed to that event in this PR, it means there are now too?